docs/src/pages/_app.page.tsx

```diff
@@ -70,8 +72,10 @@ function MyApp({ Component, pageProps }) {
     );
   }, []);
 
-  configure();
-  trackPageVisit();
+  if (stage === 'prod') {
+    configure();
+    trackPageVisit();
+  }
 
   return (
     <>
```

Comment:
Let's also limit the following scripts to prod as well:
```
<Script src="https://a0.awsstatic.com/s_code/js/3.0/awshome_s_code.js" />
<Script src="/scripts/shortbreadv2.js" />
```

Reply:
Done